gb/src/sdl_gfx_device.rs

```diff
@@ -7,33 +7,59 @@ pub struct SdlGfxDevice{
     _window_name: CString,
     renderer: *mut SDL_Renderer,
     texture: *mut SDL_Texture,
-    width:u32,
-    height:u32,
-    sacle:u8,
     discard:u8,
     turbo_mul:u8,
+    #[cfg(feature = "static_scale")]
+    screen_scale:usize,
 }
 
 impl SdlGfxDevice{
-    pub fn new(buffer_width:u32, buffer_height:u32, window_name:&str, screen_scale: u8, turbo_mul:u8, disable_vsync:bool)->Self{
+    pub fn new(window_name:&str, screen_scale: usize, turbo_mul:u8, disable_vsync:bool, full_screen:bool)->Self{
         let cs_wnd_name = CString::new(window_name).unwrap();
 
         let (_window, renderer, texture): (*mut SDL_Window, *mut SDL_Renderer, *mut SDL_Texture) = unsafe{
             SDL_Init(SDL_INIT_VIDEO);
+
+            let window_flags = if full_screen{
+                SDL_WindowFlags::SDL_WINDOW_FULLSCREEN_DESKTOP as u32
```

Comment:
What will happen when this will be enabled and the program will be compiled with static scale?

Reply:
Add warning for it